Makefile

```diff
@@ -4,7 +4,7 @@ all: test
 
 .PHONY: test
 test:
-	$(GO) test -v ./...
+	$(GO) test -v -coverprofile coverage.txt -covermode=atomic -coverpkg=antrea.io/libOpenflow/common/...,antrea.io/libOpenflow/openflow13/...,antrea.io/libOpenflow/openflow15/...,antrea.io/libOpenflow/protocol/...,antrea.io/libOpenflow/util/...  antrea.io/libOpenflow/...
```

Comment:
Maybe remove package `antrea.io/libOpenflow/openflow13`, which is not maintained in future.

Reply:
Done